Example/Backpack/BPKAppDelegate.m

```diff
@@ -38,6 +39,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
             @{NSForegroundColorAttributeName: BPKColor.textPrimaryColor};
     }
 
+//    [BPKFont setFontDefinition:[BPKRelativeFontDefinition new]];
```

Comment:
🚨

Reply:
This gets automatically un-commented by the script

Reply:
![giphy-4](https://user-images.githubusercontent.com/10763919/70636618-c4423200-1c2d-11ea-86d9-630f9578f09a.gif)